tests/manager/manager-ctor-read_preference-005.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+MongoDB\Driver\Manager::__construct(): read preference options of the wrong type will throw an invalid argmument exception
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$test = array(STANDALONE, array('readPreference' => new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_PRIMARY)));
```

Comment:
Since you're using `STANDALONE` here, the `SKIPIF` block should include `<?php NEEDS('STANDALONE'); ?>` (see other tests for examples). That said, you're not actually connecting to a remote server here, so you can simply pass `null` for `Manager::__construct()`'s first argument to fall back to the default connection string (`"mongodb://127.0.0.1/"`).